Python/ceval.c

```diff
@@ -1351,6 +1351,15 @@ eval_frame_handle_pending(PyThreadState *tstate)
         DISPATCH_GOTO(); \
     }
 
+#define DISPATCH_SAME_OPARG() \
+    { \
+        opcode = _Py_OPCODE(*next_instr); \
+        PRE_DISPATCH_GOTO(); \
+        assert(cframe.use_tracing == 0 || cframe.use_tracing == 255); \
+        opcode |= cframe.use_tracing OR_DTRACE_LINE; \
```

Comment:
I think the tracing checks here should be removed. We should be able to finish executing the opcode we started to execute (even if it's the second try) before trying to respect cframe.use_tracing.

Do reviewers agree?

Reply:
The only time it could make a difference is if one of the `_Py_Specialize_XxxXxxxx()` calls somehow managed to set the flag.